src/pages/Host.tsx

```diff
@@ -12,13 +18,27 @@ import { HostStatus } from "types/host";
 export const Host: React.FC = () => {
   const { id } = useParams<{ id: string }>();
   // Query host data
-  const { data, loading } = useQuery<HostQuery, HostQueryVariables>(GET_HOST, {
-    variables: { id },
-  });
-  const host = data?.host;
+  const { data: hostData, loading } = useQuery<HostQuery, HostQueryVariables>(
+    GET_HOST,
+    {
+      variables: { id },
+    }
+  );
+  const host = hostData?.host;
   const hostUrl = host?.hostUrl;
+  const tag = host?.tag ?? "";
   const status = host?.status as HostStatus;
 
+  // Query host data
+  const { data: hostEventData, loading: hostEventLoading } = useQuery<
+    HostEventsQuery,
+    HostEventsQueryVariables
+  >(GET_HOST_EVENTS, {
+    variables: { id, tag },
+  });
+
+  console.log(hostEventData, hostEventLoading);
```

Comment:
I think you forgot a `console.log` statement here.

Reply:
I left it there because I need to do something with hostEventData, hostEventLoading so the linter doesn't complain. I am not displaying it yet, so I'm not using it yet. The tests are still failing so it's probably not a good solution anyway. Separating this out might not be a good idea.

Reply:
`console.log` will actually pass our lint tests with a warning. You can fix your other lint tests by running `npm run codegen` with the schema changes in https://github.com/evergreen-ci/evergreen/pull/3860

Reply:
![image](https://user-images.githubusercontent.com/4605522/89578198-ca5dd200-d7ff-11ea-83bd-414c3c04ea19.png)

The linter tests fail because tag did not exist on the host type when this test was run. Rerunning it should fix that. I think Its fine to leave the console.logs() here temporarily since you plan on updating it later and users dont see this page yet, and  since they just result in linter warnings.

Reply:
awesome, thanks!